microcosm_sqlite/tests/test_stores.py

```diff
@@ -95,9 +95,9 @@ def test_count(self):
         self.populate()
 
         assert_that(self.store.count(), is_(equal_to(3)))
-        assert_that(self.store.count(offset=1), is_(equal_to(2)))
-        assert_that(self.store.count(limit=1), is_(equal_to(1)))
-        assert_that(self.store.count(offset=1, limit=1), is_(equal_to(1)))
+        assert_that(self.store.count(offset=1), is_(equal_to(3)))
```

Comment:
I'd add a comment noting that these arguments aren't actually supported, but we want to make sure that they are ignored if someone passes them in anyways?